src/grisp_onewire.erl

```diff
@@ -55,7 +74,16 @@ transaction(Fun) when is_function(Fun) ->
             erlang:raise(Class, Reason, Stacktrace)
     end.
 
-%% Spec: | S | AD,0 ‖ A ‖ DRST ‖ A ‖ Sr | AD,1 ‖ A | <byte> ‖ A\ | P |
+%% @doc <q>Performs a global reset of device state machine logic. Terminates any
+%% ongoing 1-Wire communication.</q>
+%%
+%% Command code: f0h.
```

Comment:
What is the command code used for? Why are we documenting it?

Reply:
You can find those codes in the [data sheet of the one wire master](https://datasheets.maximintegrated.com/en/ds/DS2482-100.pdf). I found it useful to see what codes are send to the one wire. The data sheet is great.

Reply:
I mean, does it add anything for the reader to add them here?